src/msw/window.cpp

```diff
@@ -3922,6 +3935,33 @@ bool wxWindowMSW::MSWCreate(const wxChar *wclass,
     return true;
 }
 
+WXHWND wxWindowMSW::MSWCreateWindowAtAnyPosition(WXDWORD exStyle, const wxChar* clName,
+                                                 const wxChar* title, WXDWORD style,
+                                                 int x, int y, int width, int height,
+                                                 WXHWND parent, wxWindowID id)
+{
+    WXHWND hWnd = ::CreateWindowEx(exStyle, clName, title, style, x, y, width, height,
+                                   parent, (HMENU)wxUIntToPtr(id), wxGetInstance(),
+                                   NULL); // no extra data
+
+    if ( !hWnd )
+    {
+        wxLogLastError(wxString::Format
+        (
+            wxT("CreateWindowEx(\"%s\", flags=%08lx, ex=%08lx)"),
+            clName, style, exStyle
+        ));
+    }
+
```

Comment:
Could you please remove this blank line between the `if` and `else` branches? This looks quite confusing IMHO.

Or, alternatively, add `return 0` above and remove the `else` below -- then the separating line would make sense.